### PR TITLE
[Commands][Thumbnails] Make batch-size configurable

### LINKED ISSUE #8800: [Command][Thumbnails] Make batch-size configurable
make batch-size for `pimcore:thumbnails:*` commands configurable. 

Currently, batch size is fixed to 50. Sometimes you want to generate many thumbnails for less than 50 assets ... this results in single-process generation, even there are hundreds of thumbnails generated...

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves #8800

### CODE REVIEW

File: bundles/CoreBundle/Command/ThumbnailsImageCommand.php [hunk 1]
```diff
@@ -113,30 +113,42 @@ protected function fetchItems(InputInterface $input): array
...
+            }
+        }
+
+        return $items;
     }
 
-    protected function runSingleCommand(string $assetId, InputInterface $input, OutputInterface $output): void
+    protected function runSingleCommand(string $item, InputInterface $input, OutputInterface $output): void
     {
+        list($assetId, $thumbnailConfigName) = explode('~~~', $item);
```
Comment (jdreesen) [inline bundles/CoreBundle/Command/ThumbnailsImageCommand.php]: Maybe add a third parameter to ensure the result will be a tuple:
```suggestion
        list($assetId, $thumbnailConfigName) = explode('~~~', $item, 2);
```

File: bundles/CoreBundle/Command/ThumbnailsVideoCommand.php [hunk 2]
```diff
@@ -76,50 +76,54 @@ protected function fetchItems(InputInterface $input): array
...
+        return $items;
     }
 
-    protected function runSingleCommand(string $assetId, InputInterface $input, OutputInterface $output): void
+    protected function runSingleCommand(string $item, InputInterface $input, OutputInterface $output): void
     {
         // disable versioning
         Version::disable();
 
+        list($assetId, $thumbnailConfigName) = explode('~~~', $item);
```
Comment (jdreesen) [inline bundles/CoreBundle/Command/ThumbnailsVideoCommand.php]:
```suggestion
        list($assetId, $thumbnailConfigName) = explode('~~~', $item, 2);
```

File: bundles/CoreBundle/Command/ThumbnailsImageCommand.php (hunk 1 above)
Comment (dvesh3) [reply bundles/CoreBundle/Command/ThumbnailsImageCommand.php]: makes sense! thanks 👍

**Comment by brusch** [comment]
@dvesh3 The preview configs are gone now with this PR, can you please check? Thanks!

**Review by brusch — CHANGES_REQUESTED** [review]
see comment above

**Comment by dvesh3** [comment]
> @dvesh3 The preview configs are gone now with this PR, can you please check? Thanks!

@brusch fixed it.

**Review by brusch — APPROVED** [review]

**Pull request merged**